src/App.jsx

```diff
@@ -14,57 +15,62 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () =>{
+  const [selectedGood, setSelectedGood] = useState('Jam');
+
+  function handleSelectedGood(good){
+    const selectedGood = goods.find((item) => item === good)
+    setSelectedGood(selectedGood)
+  }
+
+  function handleClearSelectedGood(){
+    setSelectedGood(null)
+  }
 
+  return (
+  <main className="section container">
+  {
+    selectedGood ?
     <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
+      {`${selectedGood} is selected`}
+      <button data-cy="ClearButton" onClick={handleClearSelectedGood}  type="button" className="delete ml-3" />
     </h1>
+    :
+    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+
+  }
 
     <table className="table">
       <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+        {
+          goods.map(good => (
+            <tr data-cy="Good" className={`${selectedGood === good && "has-background-success-light"}`}>
```

Comment:
The `className` attribute uses a template literal that may result in an invalid class name when `selectedGood` is not equal to `good`. Consider using a conditional operator to ensure a valid class name is always applied.